addons/prometheus/prometheus.yaml

```diff
@@ -25,8 +25,8 @@ metadata:
     # The prometheus-operator chart from prior Konvoy releases can't be upgraded to 8.10.0.
```

Comment:
we need to update the values sha on the line above to `825e9732efe486f372c12015b13db52faa7cfc42`